packages/blade/src/components/Skeleton/PulseAnimation.native.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+import Animated, {
+  cancelAnimation,
+  interpolateColor,
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated';
+import React from 'react';
+import type { SkeletonProps } from './Skeleton';
+import BaseBox from '~components/Box/BaseBox';
+import { castNativeType, makeMotionTime } from '~utils';
+import { useTheme } from '~components/BladeProvider';
+
+const AnimatedBox = Animated.createAnimatedComponent(BaseBox);
+
+const PulseAnimation = ({
+  contrast,
+  ...props
+}: { contrast: 'low' | 'high' } & SkeletonProps): React.ReactElement => {
+  const { theme } = useTheme();
+  // TODO: no token for 300ms delay
```

Comment:
Check with RK if we should add this?

Reply:
Asked, since this is not [actually a delay](https://github.com/razorpay/blade/pull/1398#discussion_r1263552442) instead just a duration offset, RK used duration tokens only. We can do that also.